frontend/src/containers/Homepage/Homepage.js

```diff
@@ -29,7 +29,7 @@ class Homepage extends Component {
       <div className="homepage container-fluid ">
         <div className="row">
           <div className="header">
-            <Header />
+            <Header showSearchBar={false} />
```

Comment:
This _might_ be unnecessary, but now that `Header` as two props, I think it could benefit from using [prop-types](https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes).

That way, we could specify that `showSearchBar` be a boolean, and then give it a default value of `false` so we don't need to specify `showSearchBar={ false }` below.

Reply:
Currently we only want it to be false on one page, and we want it to render on all the other ones. So if we set the default to be false, would we then have to specify `showSearchBar={ true }` on all the other pages?

Reply:
Yes and no... While we do need to explicitly set `showSearchBar` to `true`, `<Header showSearchBar />` is just shorthand for `<Header showSearchBar={ true } />`. So we wouldn't need to change any of the existing code.

Reply:
Ah ok this makes sense thanks! I'll go ahead and add this.